.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,188 @@
+      label: Other Firebase Component(s) in use
+      description: Which other Firebase components are being used?
+      multiple: true
+      options:
+        - AB Testing
+        - Analytics
+        - App Check
+        - App Distribution
+        - Authentication
+        - Crashlytics
+        - Database
+        - DynamicLinks
+        - Firestore
+        - Functions
+        - In-App Messaging
+        - Installations
+        - ML Model Downloader
+        - Messaging
+        - Performance
+        - Remote Config
+        - Storage
+        - All
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Additional SDKs you are using
+      description: 'What additional, non-Firebase SDKs are being used?'
+      placeholder: 'Facebook, AdMob, etc.'
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Targeted Platform(s)
+      description: What platform(s) is the issue affecting?
+      multiple: true
+      options:
+        - Apple Platforms
+        - Android
+        - Desktop
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Unity editor platform
+      description: What platform are you using the Unity editor on?
+      multiple: true
+      options:
+        - Mac
+        - Windows
+        - Linux
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Scripting Runtime
+      description: What scripting runtime is being used?
+      multiple: true
+      options:
+        - Mono
+        - IL2CPP
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Release Distribution Type
+      description: What release distribution is being used?
+      multiple: true
+      options:
+        - 'Pre-built SDK from https://firebase.google.com/download/unity'
+        - Open-source from this repo
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Relevant Log Output
+      description: |
+        Please copy and paste any relevant log output. To enable Firebase debug
+        logging, add `-FIRDebugEnabled` to your scheme's launch arguments.
+      placeholder: |
+        Paste your logs here. Please redact any personally identifiable
+        information. This will be automatically formatted into code, so no
+        need for backticks.
+      render: shell
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: 'If using CocoaPods for Apple platforms, the project''s Podfile.lock'
+      description: The `Podfile.lock` can help us debug versioning issues.
+      value: "<!--- Look below for instructions on how to share your Podfile.lock. --->\n\n<details>\n<summary>Expand <code>Podfile.lock</code> snippet</summary>\n<br>\n\n```yml\n\n\U0001F440 Replace this line with the contents of your Podfile.lock!\n\n```\n\n</details>\n"
+    validations:
+      required: false
```

Comment:
This is part of the iOS SDK template, and I guessed it may be helpful here. If not, I'm cool with removing it. LMK!